run_services.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Function to stop all processes gracefully
+function stop_all_processes {
+    echo "Stopping all processes..."
+    kill -SIGTERM $manager_pid $api_pid $dbsync_pid
+    wait $manager_pid $api_pid $dbsync_pid
+    echo "All processes stopped."
+    exit 0
+}
+
+# Register the stop_all_processes function to run on SIGTERM signal
+trap stop_all_processes SIGTERM
+
+# Start the first process
+esi-leap-dbsync create_schema &
+dbsync_pid=$!
```

Comment:
This command should be ran in the foreground, not the background. Additionally, if the command is unsuccessful (e.g. exits with a non-zero status code) the script should exit without launching the API or the manager service.